qa/truffle/contracts/Storage.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Is there a reason for why do we have this contract? AFAIK this is a test contract from Remix.